### PR TITLE
[DP-930] Introduce new command for datapool listing

### PR DESCRIPTION
**Pull request by segef**
#### Description

New command to list all Data Pools of the team.
`content-cli list data-pools -p <your-chosen-profile>`

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [x] I have updated docs if needed

### CODE REVIEW

File: src/content/manager/data-pool.manager.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import {logger} from "../../util/logger";
+import {DataPoolSlimTransport} from "../../interfaces/data-pool-manager.interfaces";
 
 export class DataPoolManager extends BaseManager {
     public static DATA_POOL_FILE_NAME_PREFIX = "data-pool_";
     private static API_URL = "/integration/api";
     private static DATA_POOL_PUSH_URL = DataPoolManager.API_URL + "/pool-import";
     private static DATA_POOL_ACTIONS_URL = DataPoolManager.API_URL + "/pools/{id}";
     private static DATA_POOL_PULL_URL = DataPoolManager.DATA_POOL_ACTIONS_URL + "/export";
+    private static DATA_POOLS_LIST_URL = DataPoolManager.API_URL + "/pools/paged";
```
Comment (fisgeci) [inline src/content/manager/data-pool.manager.ts:12]: I think these changes are not needed now.

Comment (segef) [reply src/content/manager/data-pool.manager.ts]: Reverted these changes 👍

**Review by fisgeci — DISMISSED** [review]

**Review by promeris — COMMENTED** [review]
Please manually bump the version in `package.json`, as the workflow isn't working properly at this time.

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -279,6 +279,18 @@ packages you have access to. 
...
+### List all data pools of the team
+
+With this command you can retrieve a list of all data pools within a team.
+The command takes your permissions into consideration and only lists the
+pools you have access to. 
+
+-   It is also possible to download packages in JSON format by adding '--json' option.
+
+```
+content-cli list packages -p <your-chosen-profile>
```
Comment (promeris) [inline DOCUMENTATION.md:291]:
```suggestion
content-cli list data-pools -p <your-chosen-profile>
```

**Review by promeris — COMMENTED** [review]
One small change for docs, other than that looks good.

**Review by promeris — APPROVED** [review]
Thanks for making changes. Please feel free to merge!

**Pull request merged**